### DISCUSSION TITLE
Gopro Wide & Log IDT to Davinci Wide Gamut / ACES

### DISCUSSION THREAD

**Discussion by markusfinholt** [xtremestuff/protune-transforms#7] [Q&A] [answered]
Hi, fantastic work, would it be possible to create an IDT to the scene referred workflows like DWG? I don't have the math / coding knowledge to do it myself.

**Comment by RikshaDriver** [comment]
These are already available - refer to the DCTLs under the LUT folder.

**Reply by markusfinholt** [reply]
Unless I'm not finding it, do you mean the native to dwg IDT? 

If that's the case, is "gopro native" a wider gamut than "gopro Wide"?

**Reply by RikshaDriver** [reply, marked as answer]
Native and Wide are the same sensor native values, except that WIDE has a white balance applied and Native does not.
